### ISSUE TITLE
explaination for negative weight in VLSM stat.img

### ISSUE THREAD

**Issue by yangwei1993** [dorianps/LESYMAP#38]
hello,
I noticed that there is postive and negative value in stat_img.nii in the SCCAN results. what is the difference between the postive and negative value? dose it mean that postive value is postively related to the behaviour score and nagetive value is nagetively related to the behavior score? 

Besides, I noticed that values of stat_im.nii is normalized from the raw_data. I don't know the formulation which translate the raw_data to stat_img data. so, it is difficult to interpret the postive and negative weight values.

**Comment by dorianps** [comment]
Yes, in theory the relationship should be positive for positive values, but
cannot promise that with 100% certainty because the developer of the
original function was skeptical of interpreting values or signs initially.
This said, I think several groups have used the sign of the weights to
interpret findings and I don't know of any anomalous issue.

The code that converts raw weights to stats is here:

https://github.com/dorianps/LESYMAP/blob/master/R/lsm_sccan.R#L213-L231

Basically, the small weights below 10% of max value are removed, and there
are a lot of voxels like that. The flipping of values also happens between
those lines.

Dorian
